README.md

```diff
@@ -109,11 +109,14 @@ When constructing a whole Project from a directory, you can choose to link all
 dependencies instead of copying them in as Projects:
 
 ```js
-let project = Project.fromDir("./sample-project", { linkDeps: true });
+let project = Project.fromDir('./sample-project', { linkDeps: true });
 project.files['extra.js'] = '// stuff';
 project.write();
 ```
 
 This will generate a new copy of sample-project, with symlinks to all its
 original dependencies, but with "extra.js" added.
 
+By default, `linkDeps` will only link up `dependencies` (which is appropriate
+for libraries). If you want to also include `devDependencies` (which is
+appropriate for apps) you can use `linkDevDeps` instead.
```

Comment:
It's implemented so that `linkDevDeps` also implies `linkDeps` (I don't think there's any scenario where you wouldn't want that), but it's also harmless to pass both.